app/models/authentication.rb

```diff
@@ -81,6 +81,18 @@ def raise_event(status, _message = nil)
     MiqEvent.raise_evm_event_queue(ci, "#{prefix}_auth_#{status}")
   end
 
+  def assign_values(options)
```

Comment:
Great! This method seems pointless now...the caller can just call `attributes=` directly, so let's remove it.